FlaxEditor/Surface/VisjectSurface.Serialization.cs

```diff
@@ -96,109 +96,109 @@ private static void ReadCommonValue(BinaryReader stream, ref object value)
-            {
-                value = new Vector4(stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle());
-            }
-                break;
-            case 6: // CommonType::Color:
-            {
-                value = new Color(stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle());
-            }
-                break;
-            case 7: // CommonType::Guid:
-            {
-                value = new Guid(stream.ReadBytes(16));
-            }
-                break;
-            case 8: // CommonType::String:
-            {
-                int length = stream.ReadInt32();
-                if (length <= 0)
-                {
-                    value = string.Empty;
-                }
-                else
-                {
-                    var data = new char[length];
-                    for (int i = 0; i < length; i++)
-                    {
-                        var c = stream.ReadUInt16();
-                        data[i] = (char)(c ^ 953);
-                    }
-                    value = new string(data);
-                }
-                break;
-            }
-            /*case 9:// CommonType::Box:
-            {
-                BoundingBox v;
-                ReadBox(&v);
-                data.Set(v);
-            }
-                break;
-            case 10:// CommonType::Rotation:
-            {
-                Quaternion v;
-                ReadQuaternion(&v);
-                data.Set(v);
-            }
-                break;
-            case 11:// CommonType::Transform:
-            {
-                Transform v;
-                ReadTransform(&v);
-                data.Set(v);
-            }
-                break;
-            case 12:// CommonType::Sphere:
-            {
-                BoundingSphere v;
-                ReadSphere(&v);
-                data.Set(v);
-            }
-                break;
-            case 13:// CommonType::Rect:
-            {
-                Rect v;
-                ReadRect(&v);
-                data.Set(v);
-            }
-                break;*/
-            case 15: // CommonType::Matrix
-            {
-                value = new Matrix(stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(),
-                                   stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(),
-                                   stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(),
-                                   stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle(), stream.ReadSingle());
-                break;
-            }
-
-            default: throw new SystemException();
+            {
+                case 0: // CommonType::Bool:
+                    value = stream.ReadByte() != 0;
+                    break;
+                case 1: // CommonType::Integer:
+                    {

```

Comment:
The standard C# Visual Studio formatter changes the formatting of lines like those.